Plugins/ExternalTexture/Source/ExternalTexture_OpenGL.cpp

```diff
@@ -6,9 +6,121 @@
 #include <napi/pointer.h>
 #include <bx/bx.h>
 #include <bgfx/bgfx.h>
+#include <GL/gl.h>
 
 #include "ExternalTexture_Base.h"
 
+namespace
+{
+    struct TextureFormatInfo
+    {
+        GLint m_fmt;
+        GLint m_fmtSrgb;
+    };
+
+    static TextureFormatInfo s_textureFormat[] =
```

Comment:
This table is usually derived from bgfx. It would be better to start from here: https://github.com/BabylonJS/bgfx/blob/master/src/renderer_gl.cpp#L212-L325. Note this bgfx is a fork, but is only intended to be a staging area for changes.